react-backend/client/src/components/visitors/main.js

```diff
@@ -43,7 +43,7 @@ export default class Main extends Component {
     'Content-Type': 'application/json',
   });
 
-  checkUserExists = (e) => {
+  userCheckExists = (e) => {
```

Comment:
Why is this a better name than before? Before the method name began with a verb, and had an instructional voice. The new one sounds like the user is doing something, which is not the case.

Reply:
Because it matches the db query functions/files. Lots of similar ones are prefixed with _user_ so didn't want to change the word order.

Let me know if you think it's actually better the previous way and I can change it back. Just trying to create some order 😕

Reply:
Don't think we need to match method names in the front end with file name in the backend. That's too much coupling IMO. The contract is the API.

Reply:
@eliascodes not sure I'm following... Won't this make me loose the work that I've done since? Or will I need to stash this first then `git reset [branch]`?

Reply:
If you have done work since, yes you will lose those commits (although you will keep the changes in your working copy as long as you don't use `--hard`). tbh not a big deal either way, just tidier to not have commits that are not meaningful. You can always solve it after the fact with an interactive rebase :wink: